.proxyrc.json

```diff
@@ -1,5 +1,5 @@
 {
   "/api": {
-    "target": "http://localhost:3000/"
+    "target": "http://127.0.0.1:3000/"
```

Comment:
Note to reviewers: this is for [Apple arm processors compat](https://stackoverflow.com/a/74464534) (dunno why and I don't want to spend the required time to fully understand why)